server/msteams/client_test.go

```diff
@@ -0,0 +1,115 @@
+package msteams
+
+import (
+	"testing"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/testutils"
+	"github.com/stretchr/testify/assert"
+	msgraph "github.com/yaegashi/msgraph.go/beta"
+)
+
+func TestConvertToMessage(t *testing.T) {
+	teamsUserID := testutils.GetTeamUserID()
+	teamsUserDisplayName := "mockTeamsUserDisplayName"
+	teamsReplyID := "mockTeamsReplyID"
+	content := "mockContent"
+	subject := "mockSubject"
+	reactionType := "mockReactionType"
+	attachmentContent := "mockAttachmentContent"
+	attachmentContentType := "mockAttachmentContentType"
+	attachmentName := "mockAttachmentName"
+	attachmentURL := "mockAttachmentURL"
+	assert := assert.New(t)
+	resp := convertToMessage(&msgraph.ChatMessage{
```

Comment:
I would like to see another testcase where the data passed is almost empty (a lot of nils) to see that the application doesn't panic and is following the other paths in the ifs.